WooCommerce/Classes/POS/PointOfSaleNotificationScheduler.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+import UserNotifications
+import Yosemite
+import Experiments
+
+// periphery: ignore - work in progress
+protocol PointOfSaleNotificationScheduling {
+    func scheduleLocalNotificationIfEligible(for merchantType: PointOfSaleNotificationScheduler.MerchantType) async
+}
+
+// periphery: ignore - work in progress
+final class PointOfSaleNotificationScheduler: PointOfSaleNotificationScheduling {
```

Comment:
nit; We haven't renamed everything yet, but I think we decided to call things with `POS` prefix

```suggestion
final class POSNotificationScheduler: POSNotificationScheduling {
```